data/presets/historic/memorial/blue_plaque-GB-IE.json

```
@@ -0,0 +1,28 @@
+{
+    "icon": "temaki-plaque",
+    "fields": [
+        "name",
+        "inscription",
+        "subject/wikidata"
+    ],
+    "moreFields": [
+        "direction",
+        "image",
+        "wikimedia_commons"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "historic": "memorial",
+        "memorial": "blue_plaque"
+    },
```

I thinks this will make it that the preset will trigger on the secondary (but unique) tag and should the other tag be missing, trigger the "missing tag" warning to add it.

I will commit this right away to test it.

```suggestion
    "tags": {
        "memorial": "blue_plaque"
    },
    "addTags": {
        "historic": "memorial",
    },
```